string drive_mode			
int32 drive_status			
int32 spray_status			
int32 error_status			
